Update readme (#64)
vercel/fetch: readme.md

# Fetch Monorepo

> This is the official Vercel fetch monorepo

Reviewer: ```suggestion
This fetch monorepo contains three packages:

- `@vercel/fetch`
- `@vercel/fetch-retry`
- `@vercel/fetch-cached-dns`
```

## Getting Started

> There are a variety of ways to use these packages. The core package `@vercel/fetch` bundles the remaining packages together into a super powered [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) client. By default this package will use its peer dependency [node-fetch](https://github.com/node-fetch/node-fetch), but it also supports other fetch implementations.

Reviewer: ```suggestion
`@vercel/fetch` bundles all packages inside this monorepo together into a super-powered [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) client. By default, this package will use its peer dependency [node-fetch](https://github.com/node-fetch/node-fetch), but it also supports other fetch implementations.
```

# Fetch Monorepo

This fetch monorepo contains three packages:

- `@vercel/fetch`
- `@vercel/fetch-retry`
- `@vercel/fetch-cached-dns`

> These packages are recommended to be used in Node.js environments. It aims to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperale between both browser and server environments.

Reviewer 2: ```suggestion
These packages are recommended to be used in Node.js environments. It aims to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperable between both browser and server environments.
```

> These packages are recommended to be used in Node.js environments. It aims to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperable between both browser and server environments.

Reviewer 2: ```suggestion
These packages are designed for use with Node.js in order to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperable between both browser and server environments.
```
